tests/suite.py

```diff
@@ -1,12 +1,13 @@
 """Module for loading all tests, so we can run them from setup.py"""
-import os
+import sys
+from os.path import dirname
 
-try:
+# this module has to stand alone, so we can't use tests.py2
+if sys.version_info < (2, 7):
     import unittest2 as unittest
-    unittest  # quiet "redefinition of unused ..." warning from pyflakes
-except ImportError:
+else:
     import unittest
```

Comment:
It doesn't seem to be able to find `tests.py2` when run from `setup.py`. Not sure how that works exactly, but didn't want to wrestle with it.